### PR TITLE
Feat/public private status

### LINKED ISSUE #552: List issue status - public or private
List and record the issue status (public or private) on the task page and if it is a bounty.

The part to display if this issue is a bounty is just listed static, cause we just have now bounties.

### Why
We have public and private issues, so we should display the current status.

### Level
Beginner

### Screenshot attached
Screenshot 2020-03-23 at 21 32 39

### How to reproduce the user scenario
 - Create a user
 - Import an issue
 - The status should be displayed as the screenshot

### PR DESCRIPTION
**Pull request by ShubhamCanMakeCommit**
## Description

> Displays Bounty, Private or Public statuses as Icons.

## Issue

> Closes #552 

## Impacted Area

> Task page

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

- Create / log user
- Click Explore Tasks
- Open a task and you should see the changes

image

### CODE REVIEW

File: frontend/src/components/task/task-status-icons.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  LockOpen as PublicIcon
+} from '@material-ui/icons'
+import {
+  Paper,
+  Typography
+} from '@material-ui/core'
+
+const styles = {
+  IconStyle: {
+    fontSize: 80
```
Comment (alexanmtz) [inline frontend/src/components/task/task-status-icons.js:15]: @ShubhamCanMakeCommit can we reduce the icon to 48?
```suggestion
    fontSize: 48
```

**Comment by ShubhamCanMakeCommit** [comment]
image

File: frontend/src/components/task/task-status-icons.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        elevation={ 0 }
+        children={
+          <React.Fragment>
+            <Typography type='subheading' variant='caption' style={ {
+              padding: 10,
+              color: 'gray',
+              textAlign: 'center',
+              fontSize: 30
+            } }>
+                      Task is
```
Comment (alexanmtz) [inline frontend/src/components/task/task-status-icons.js:75]: @ShubhamCanMakeCommit We need to apply the internationalization here as we use ReactIntl: https://github.com/formatjs/react-intl, and please check examples of use in our platform

File: frontend/src/components/task/task-status-icons.js [hunk 3]
```diff
@@ -52,7 +57,7 @@ const TaskStatusIcons = ({ status, bounty }) => {
           }
           <Typography
             style={ styles.fontStyle }>
-            { (status === 'private') ? 'Private' : 'Public' }
+            <FormattedMessageFixed id='task.status.icon.text' defaultMessage={ Status } />
```
Comment (alexanmtz) [inline frontend/src/components/task/task-status-icons.js:60]: You need to create a key for each status, this solution will not work properly.

File: frontend/src/components/task/task-status-icons.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+      return (
+        <React.Fragment>
+          { (status === 'private')
+            ? (<PrivateIcon style={ styles.IconStyle } />)
+            : (<PublicIcon style={ styles.IconStyle } />)
+          }
+          <Typography
+            style={ styles.fontStyle }>
+            { (status === 'private') ? <FormattedMessage id='task.private.icon.text' defaultMessage='Private' />
+              : <FormattedMessage id='task.public.icon.text' defaultMessage='Public' />
```
Comment (alexanmtz) [inline frontend/src/components/task/task-status-icons.js:57]: We can't have same ids, they need different names like `task.private.icon.text.public` and `task.private.icon.text.private`, something like that.

File: frontend/src/translations/en.json [hunk 5]
```diff
@@ -129,6 +129,10 @@
   "task.assignment.message.user": "Send a message to this user",
   "task.assignment.message.reason": "Send a message to this user interested if you have questions and if you want to know more details about how your issue will be solved.",
   "task.assignment.message.send": "Send message",
+  "task.bounty.icon.text":"Bounty",
+  "task.private.icon.text":"Private",
+  "task.public.icon.text":"Public",
```
Comment (alexanmtz) [inline frontend/src/translations/en.json:134]: @ShubhamCanMakeCommit same here

File: frontend/src/translations/br.json [hunk 6]
```diff
@@ -109,6 +109,10 @@
   "task.assignment.remove.reason": "Qual o motivo dele não trabalhar mais nesta tarefa?",
   "task.assignment.actions.cancel": "Cancelar",
   "task.assignment.actions.apply": "Aplicar",
+  "task.bounty.icon.text":"Recompensa",
+  "task.private.icon.text":"Privado",
+  "task.public.icon.text":"Público",
```
Comment (alexanmtz) [inline frontend/src/translations/br.json:114]: @ShubhamCanMakeCommit and here

File: frontend/src/components/task/task-status-icons.js (hunk 4 above)
Comment (ShubhamCanMakeCommit) [reply frontend/src/components/task/task-status-icons.js:57]: They are different only. task.**private**.icon.text and task.**public**.icon.text.

Am I correct ? Or maybe I am not getting what you mean to say.  You can maybe write the exact code.

**Pull request merged**